packages/databricks-vscode/src/language/notebooks/DatabricksNbCellLimits.ts

```diff
@@ -0,0 +1,28 @@
+import {workspaceConfigs} from "../../vscode-objs/WorkspaceConfigs";
+import {FileUtils} from "../../utils";
+import {Uri} from "vscode";
+import {ConnectionManager} from "../../configuration/ConnectionManager";
+
+export async function setDbnbCellLimits(
```

Comment:
we are only setting the cell limits here right? Where are we parsing?

Reply:
This enables vscode to parse the notebooks correctly. Only `# MAGIC` cells do not work right now. They will be handled in the python init script in a separate PR.